packages/redux/src/users/personalIds/types/action.types.ts

```diff
@@ -28,6 +28,28 @@ export type FetchPersonalIdsAction =
   | FetchPersonalIdsFailureAction
   | FetchPersonalIdsSuccessAction;
 
+//
+// Fetch personal id
+//
+export interface FetchPersonalIdFailureAction extends Action {
```

Comment:
Nitpicking: It would be better if it was renamed to `FetchUserPersonalIdFailureAction` to be consistent. This applies for all action types here, not just for the failure one.

Reply:
done :)